package.json

```diff
@@ -49,6 +49,7 @@
     "eslint-plugin-es5": "1.2.0",
     "eslint-plugin-react": "7.7.0",
     "express": "4.16.3",
+    "@geo-maps/countries-coastline-10km": "0.6.0",
```

Comment:
Is the `countries-coastline-10km` package necessary if the actual JSON file has been copied under `world-base.geo.json`? Maybe we could just add credits to the readme.

Reply:
I would go the other way. 
For more fluid maintenance in time. The JSON should be copied automatically on build or used strait from it's original location. 

Im not much used to nodejs but this seem the way to go. Although I woudn't know what would be the best strategy. So I did not push anything on this part exept adding the source package. 

It's an improvement but an incomplete feature. Maybe it should come with the opening of an issue.

Reply:
What would be the best thing to do with this ? 

Im keeping it in my fork untill it's a full feature. 

You accept it like that and I open an issue about it on your repo.  

Or I remove the dependency, add credits to the readme and you will accept it.